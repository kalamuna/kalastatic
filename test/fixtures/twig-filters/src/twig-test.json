{
  "description": "This is a metadata file that's injected into twig-test!",
  "person": "@kstat/person.json"
}
