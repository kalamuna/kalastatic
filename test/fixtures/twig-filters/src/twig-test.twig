---
title: Hello World!
---
<h1>{{ title }}</h1>
<p>{{ description }}</p>

<div class="twig-test">
  {{title|slugifythis}}
</div>

<h2 class="globalVar">{{ awesomeSauce }}</h2>

<dl>
  <dt>build_path</dt>
  <dd><pre>{{ build_path }}</pre></dd>
<dl>

Hello, {{person.name}}!

This is a class: <div class="{{person.name|slug}}">Class!</div>
