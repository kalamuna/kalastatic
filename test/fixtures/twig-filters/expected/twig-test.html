<h1>Hello World!</h1>
<p>This is a metadata file that's injected into twig-test!</p>

<div class="twig-test">
  hello-world
</div>

<h2 class="globalVar">This Is The AwesomeSauce!</h2>

<dl>
  <dt>build_path</dt>
  <dd><pre>/</pre></dd>
<dl>

Hello, World!

This is a class: <div class="hello-world">Class!</div>
